lib/oppen/print_stack.rb

```diff
@@ -64,6 +64,7 @@ def initialize(width, new_line, config, space, out)
     #
     # @return [String]
     def output
+      buffer.truncate(buffer.pos)
       buffer.string
     end
```

Comment:
Yes and no. We need it for the case where we do not overide enough chars to erase the content we erased from the buffer. It will only matter it we used `seek`, so no need for a guard.

Reply:
Ok so we explored together and I missed the fact that it's the `output` function, and since we introduced the `erase` function, we're now forced to do this all the time.

If no erasure occured, this will basically do nothing.